docs/backuprestore-with-oadp.md

```diff
@@ -431,7 +437,99 @@ spec:
 > 1. The examples provided are just for reference. Create your own backup and restore CRs based on your needs.
 > 2. The backup and restore CRs must be created in the same namespace where the OADP is installed which is `openshift-adp`.
 > 3. The OADP configmaps should only contain backup and restore CRs which are in YAML format or multiple document YAML format. Other type of CRs are unknown to the LCA and will be ignored.
->
+
+##### 1.1 Backup resource when LVMS is used
+
+When LVMS the selected provider to provide dynamic storage in the cluster, the backup and restore resources for the 
```

Comment:
nit, reword," When LVMS is used to provide dynamic storage in the cluster "

Reply:
thanks